.circleci/config.yml

```
@@ -0,0 +1,48 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: python:3.7
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-py3-apt-{{ checksum "setup.py" }}
```

Ajouter le nom de la branche au cas où, pour éviter une différence de version de dépendance en cours de travail ? 🤔 
```suggestion
          key: v1-py3-apt-{{ .Branch }}-{{ checksum "setup.py" }}
```